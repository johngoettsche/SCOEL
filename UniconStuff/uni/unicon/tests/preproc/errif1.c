/* error message test: erroneous expression on #if */

#if 1+2 <
#endif
