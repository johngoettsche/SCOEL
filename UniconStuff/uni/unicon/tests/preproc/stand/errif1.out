pp: File errif1.c; Line 3: syntax error in condition of #if
